products/models.py

```diff
@@ -55,3 +54,10 @@ class ProductLike(models.Model):
     class Meta:
         db_table = 'products_likes'
 
+class ProductStatus(models.Model):
+    product  = models.ForeignKey(Product, on_delete=models.CASCADE)
+    creating = models.BooleanField(null=True)
+    created  = models.BooleanField(null=True)
+
+    class Meta:
+        db_table = 'products_status'
```

Comment:
아래와 같이 수정해주세요. product_status table에 저장되는 값이 `PENDING`, `REGISTERED`, `OPEN` 등이 저장되도록 해주세요.
결론적으로 product가 status를 참조하고 있고, 임시저장 중이던 파일을 완료하면 product가 참조하고 있던 status를 PENDING -> REGISTERD로 수정해주는 것이죠.

```suggestion
class ProductStatus(models.Model):
    status = models.BooleanField(null=True)

    class Meta:
        db_table = 'product_status'
```
```python
class Product(TimeStampedModel):
    status = models.ForeignKey('ProductStatus')
    ..
```

Reply:
아하 감사합니다 병면님 !  수정해서 올렸습니다 !